### PR TITLE
[zh] Sync #14228 1.20 performance test into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #14228 1.20 performance test into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/ops/deployment/performance-and-scalability/index.md [hunk 1]
```diff
@@ -113,32 +113,29 @@ Envoy 代理在向客户端发送响应后收集原始遥测数据。
...
 
-{{< image width="90%"
-    link="latency_p99_fortio_with_jitter.svg"
+<img width="90%" style="display: block; margin: auto;"
+    src="latency-p99-fortio-with-uniform.svg"
     alt="P99 延迟 vs 客户端连接"
     caption="P99 延迟 vs 客户端连接"
->}}
+/>
+<p><h2 style="text-align: center;"> P99 延迟 vs 客户端连接 </h2></p>
```
Comment (windsonsea) [inline content/zh/docs/ops/deployment/performance-and-scalability/index.md:139]: The following lines 140 - 143 can be removed, which has an inconsistent style with other zh docs.

File: content/zh/docs/ops/deployment/performance-and-scalability/index.md [hunk 2]
```diff
@@ -113,32 +113,29 @@ Envoy 代理在向客户端发送响应后收集原始遥测数据。
...
 [Istio 基准测试](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)获得了这些结果，
-测试标准是每秒 1000 请求，负载为 1KB，使用了 16 个客户端连接和 2 个代理 worker 并启用双向 TLS。
+测试标准是每秒 1000 请求，负载为 1KB，使用了 2、4、8、16、32、64 个客户端连接和 2 个代理 worker 并启用双向 TLS。
+
+注意：不同的硬件会给出不同的值。
 
-{{< image width="90%"
-    link="latency_p90_fortio_with_jitter.svg"
+<br><img width="90%" style="display: block; margin: auto;"
+    src="latency-p90-fortio-with-uniform.svg"
```
Comment (windsonsea) [inline content/zh/docs/ops/deployment/performance-and-scalability/index.md:127]: 这 svg 上的几个文字要翻吗

File: content/zh/docs/ops/deployment/performance-and-scalability/index.md [hunk 3]
```diff
@@ -113,32 +113,29 @@ Envoy 代理在向客户端发送响应后收集原始遥测数据。
...
     alt="P90 延迟 vs 客户端连接"
     caption="P90 延迟 vs 客户端连接"
->}}
+/>
+<p><h2 style="text-align: center;"> P90 延迟 vs 客户端连接 </h2></p><br>
 
-{{< image width="90%"
-    link="latency_p99_fortio_with_jitter.svg"
+<img width="90%" style="display: block; margin: auto;"
+    src="latency-p99-fortio-with-uniform.svg"
```
Comment (windsonsea) [inline content/zh/docs/ops/deployment/performance-and-scalability/index.md:134]: ditto

File: content/zh/docs/ops/deployment/performance-and-scalability/index.md (hunk 2 above)
Comment (wilsonwu) [reply content/zh/docs/ops/deployment/performance-and-scalability/index.md:127]: 我看了一下不好操作，这个 svg 名为 svg，实际上还是一整张图片，文字没有被抽离出来，所以我就没弄。

Comment (windsonsea) [reply content/zh/docs/ops/deployment/performance-and-scalability/index.md:127]: Ok, all right. 后续我尝试抽离一下文字，变成可编辑的

File: content/zh/docs/ops/deployment/performance-and-scalability/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/docs/ops/deployment/performance-and-scalability/index.md:118]: Done, removed all English content in comments.

**Review by windsonsea — APPROVED** [review]
thanks, lgtm

**Pull request merged**